### PR TITLE
remove version requirement from etcd, shouldn't be needed anymore

### PR DESCRIPTION
**Pull request by maxamillion**

### CODE REVIEW

File: roles/etcd/tasks/main.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@
   when: "'ipv4' not in hostvars[inventory_hostname]['ansible_' ~ etcd_interface] or 'address' not in hostvars[inventory_hostname]['ansible_' ~ etcd_interface].ipv4"
 
 - name: Install etcd
-  action: "{{ ansible_pkg_mgr }} name=etcd-2.* state=present"
+  action: "{{ ansible_pkg_mgr }} name=etcd* state=present"
```
Comment (sdodson) [inline roles/etcd/tasks/main.yml]: Not sure why my comment was in a weird place, but I'd say drop the * too.

**Comment by maxamillion** [comment]
Fixed :)

**Comment by abutcher** [comment]
aos-ci-test

**Comment by abutcher** [comment]
@brenton :incoming_envelope:

**Pull request merged**